src/pkg/selectors/sharepoint.go

```diff
@@ -390,14 +390,14 @@ const (
 	SharePointCategoryUnknown sharePointCategory = ""
 
 	// types of data identified by SharePoint
-	SharePointWebURL      sharePointCategory = "SharePointWebURL"
-	SharePointSite        sharePointCategory = "SharePointSite"
-	SharePointList        sharePointCategory = "SharePointList"
-	SharePointListItem    sharePointCategory = "SharePointListItem"
-	SharePointLibrary     sharePointCategory = "SharePointLibrary"
-	SharePointLibraryItem sharePointCategory = "SharePointLibraryItem"
-	SharePointPageFolder  sharePointCategory = "SharePointPageFolder"
-	SharePointPage        sharePointCategory = "SharePointPage"
+	SharePointWebURL        sharePointCategory = "SharePointWebURL"
+	SharePointSite          sharePointCategory = "SharePointSite"
+	SharePointList          sharePointCategory = "SharePointList"
+	SharePointListItem      sharePointCategory = "SharePointListItem"
+	SharePointLibraryFolder sharePointCategory = "SharePointLibraryFolder"
```

Comment:
is this part of the selector persisted or used for metrics? If it is, do we need to worry about backwards compatibility for it?

If we can't change the enum value that's alright as long as we know the correspondence. Changing the enum name will at least help